test/integration_testcase.go

```diff
@@ -118,7 +140,14 @@ func getFile(path string) (bool, string, error) {
 	if err != nil {
 		return false, "", nil
 	}
-	f, err := ioutil.ReadFile(path)
+
+	// Ensure it's the absolute, symlink-less path we're returning
+	abs, err := filepath.EvalSymlinks(path)
+	if err != nil {
+		return false, "", nil
+	}
+
+	f, err := ioutil.ReadFile(abs)
```

Comment:
Is this change actually needed? I believe `ioutil.ReadFile()` will traverse symlinks itself, if needed.

Or is it just so that we can differentiate the return values with the first parameter?

I realize this function isn't being introduced in this PR, but let's please add at least some named return values in order to clarify what's happening with those values: `(exists bool, contents string, err error)`